tsconfig.json

```diff
@@ -1,21 +1,18 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es2017",
-        "sourceMap": true,
-        "allowJs": false,
-        "moduleResolution": "node",
-        "strict": true,
-        "declaration": true,
-        "downlevelIteration": true,
-        "baseUrl": ".",
-        "esModuleInterop": true,
-        "resolveJsonModule": true,
-        "outDir": "./lib",
-        "types": [
-            "jest",
-            "node"
-          ]
-    },
-    "include": ["./src"]
-}
\ No newline at end of file
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
```

Comment:
Is there a reason we need to downgrade our target?

Reply:
This isn't required it is just to be consistent across all our TS libraries which are using ES6